modules/rosa-creating-cluster.adoc

```diff
@@ -20,6 +20,21 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 
 .Procedure
 
+. Enable ROSA to create JSON files for account-wide roles and policies, including Operator policies. This is supported for the `rosa sts-init --json` mode only:
++
+[source,terminal]
+----
+$ rosa create account-roles --json --version-4.7
+----
+This would provide a simple way to generate the account wide roles and policies.
+
+. Enable ROSA to create cluster-specific roles. This option takes in a parameter of the cluster name or ID and then outputs the relevant JSON file and sample command to create the OIDC provider, including the thumbprint. The roles created include the relevant prefix for the cluster name.
```

Comment:
This is now split into two separate commands:
- `rosa create operator-roles`
- `rosa create oidc-provider`